CAIPs/caip-24.md

```diff
@@ -0,0 +1,70 @@
+---
+caip: 24
+title: Asset Reference for the cosmos Asset Namespace
+author: Theobold Frazier
+discussions-to: https://github.com/shapeshift/CAIPs/issues/1
```

Comment:
The PR will need to be self-referenced when opening it in ChainAgnostic/CAIPs

Reply:
Good call.